client/state/automated-transfer/middleware.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Internal dependencies
+ */
+import localforage from 'lib/localforage';
+import {
+	THEME_TRANSFER_STATUS_RECEIVE,
+	AUTOMATED_TRANSFER_STATUS_SET,
+	AUTOMATED_TRANSFER_ELIGIBILITY_UPDATE
+} from 'state/action-types';
+
+const clearBrowserStorageAndRefresh = ( dispatch, { status } ) => {
+	if ( typeof window !== 'undefined' && status === 'complete' ) {
+		localStorage.clear();
+		localforage.clear().then(
+			() => window.location.reload()
```

Comment:
<img width="616" alt="screen shot 2017-01-27 at 11 25 49 am" src="https://cloud.githubusercontent.com/assets/1270189/22384608/f7d9c478-e483-11e6-9ae0-8ff5032aaf86.png">

It's required in this case. The alternative would be:
```
localForage.clear().then( window.location.reload.bind( window.location ) );
```